src/MPoly.jl

```diff
@@ -570,7 +570,7 @@ function show(io::IO, ::MIME"text/plain", p::MPolyRing)
   print(io, " in ", ItemQuantity(nvars(p), "variable"), " ")
   if n > max_vars
     join(io, symbols(p)[1:max_vars - 1], ", ")
-    println(io, "..., ", symbols(p)[n])
+    println(io, ", ..., ", symbols(p)[n])
```

Comment:
This seems right.

However, the exact same code appears in multiple places, so perhaps we should fix them all at once?
```
src/AbsMSeries.jl:125:    println(io, "..., ", symbols(p)[n])
src/FreeAssAlgebra.jl:73:    println(io, "..., ", symbols(a)[n])
src/LaurentMPoly.jl:42:    println(io, "..., ", symbols(p)[n])
src/MPoly.jl:573:    println(io, "..., ", symbols(p)[n])
```

Actually at least two of these have further issues:
```patch
diff --git a/src/AbsMSeries.jl b/src/AbsMSeries.jl
index 33782bf97..bf3811db2 100644
--- a/src/AbsMSeries.jl
+++ b/src/AbsMSeries.jl
@@ -119,7 +119,7 @@ function show(io::IO, ::MIME"text/plain", p::MSeriesRing)
   max_vars = 5 # largest number of variables to print
   n = nvars(p)
   print(io, "Multivariate power series ring")
-  print(io, "in ", ItemQuantity(nvars(p), "variable"), " ")
+  print(io, " in ", ItemQuantity(nvars(p), "variable"), " ")
   if n > max_vars
     join(io, symbols(p)[1:max_vars - 1], ", ")
     println(io, "..., ", symbols(p)[n])
diff --git a/src/LaurentMPoly.jl b/src/LaurentMPoly.jl
index 93516b526..c8a6b35e2 100644
--- a/src/LaurentMPoly.jl
+++ b/src/LaurentMPoly.jl
@@ -36,7 +36,7 @@ function show(io::IO, ::MIME"text/plain", p::LaurentMPolyRing)
   max_vars = 5 # largest number of variables to print
   n = nvars(p)
   print(io, "Multivariate Laurent polynomial ring")
-  print(io, "in ", ItemQuantity(nvars(p), "variable"), " ")
+  print(io, " in ", ItemQuantity(nvars(p), "variable"), " ")
   if n > max_vars
     join(io, symbols(p)[1:max_vars - 1], ", ")
     println(io, "..., ", symbols(p)[n])
```

I guess we should add some tests for those types that test the actual output, just like we do for maps in the "Generic.Map.printing" testset.

Reply:
I've made some adjustments to this PR but did not yet attempt to add a dedicated test.